_data/tagging-status.yml

```diff
@@ -6543,9 +6543,9 @@
    status: currently-incompatible
    included-in: [tlc3]
    priority: 2
-   comments: overwrites tabular/array commands
-   issues: [87]
-   updated: 2024-07-06
+   comments: "`ltj<name>`, `ltjt<name>`, and `ltjs<name>` classes incompatible."
+   issues: [87,713]
```

Comment:
if #87 is resolved we should perhaps stop listing it

Reply:
Should be removed now